patch.diff

```diff
@@ -443,135 +158,74 @@ diff -raN -U 6 original/moba-desktop/src/utils/environment.js modified/moba-desk
-             experimentalFeatures: false,
-             enableRemoteModule: true,
-             autoplayPolicy: 'user-gesture-required',
-         },
-     });
-+    require("@electron/remote/main").enable(overlayWindow.webContents);
-+
-     navigation_1.initNavigationSecurity(overlayWindow);
-     return overlayWindow;
- };
--//# sourceMappingURL=game-overlay.window.js.map
-\ No newline at end of file
-+//# sourceMappingURL=game-overlay.window.js.map
-diff -raN -U 6 original/moba-desktop/src/windows/main/main.preload.js modified/moba-desktop/src/windows/main/main.preload.js
---- original/moba-desktop/src/windows/main/main.preload.js	2022-02-02 18:14:04.024680891 +0000
-+++ modified/moba-desktop/src/windows/main/main.preload.js	2022-02-02 18:14:07.874680746 +0000
-@@ -24,7 +24,7 @@
- const environment_1 = require("../../../../moba-web/src/common/utils/environment");
- const desktop_types_1 = require("../../../../moba-web/src/common/types/desktop/desktop.types");
- window.__DESKTOP__BRIDGE__ = desktop_bridge_1.bridge;
- window.__APP_ENV__ = environment_1.serializeAppConfig(appConfig, { APP_IS_DESKTOP: 'true' });
- window.__WINDOW_TYPE__ = desktop_types_1.DesktopWindowType.MAIN;
- console.info('App Logs: ', desktop_bridge_1.bridge.logsPath);
--//# sourceMappingURL=main.preload.js.map
-\ No newline at end of file
-+//# sourceMappingURL=main.preload.js.map
-diff -raN -U 6 original/moba-desktop/src/windows/main/main.window.js modified/moba-desktop/src/windows/main/main.window.js
---- original/moba-desktop/src/windows/main/main.window.js	2022-02-02 18:14:04.024680891 +0000
-+++ modified/moba-desktop/src/windows/main/main.window.js	2022-02-02 18:14:07.874680746 +0000
-@@ -73,16 +73,18 @@
-                 experimentalFeatures: false,
-                 enableRemoteModule: true,
-                 autoplayPolicy: 'user-gesture-required',
-             },
-             show: false,
-             resizable: true,
-+            icon: path.join(__dirname, "../../../../../resources/icons/512x512.png"),
+ exports.getMainWindow = getMainWindow;
+-function ensureMainWindow() {
++async function ensureMainWindow() {
+     logs_1.generalLog.debug('Ensure main window');
+     if (!mainWindow) {
+         const { width: settingsWidth, height: settingsHeight } = root_store_1.getAppSettings().getDefaultAppWindowSize();
+         const { width: fallbackWidth, height: fallbackHeight } = electron_1.screen.getPrimaryDisplay().workAreaSize;
+         const defaultWidth = Math.min(lang_1.isDef(settingsWidth) ? settingsWidth : fallbackWidth, fallbackWidth);
+         const defaultHeight = Math.min(lang_1.isDef(settingsHeight) ? settingsHeight : fallbackHeight, fallbackHeight);
+         appWindowState = electron_window_state_1.default({
+             defaultWidth,
+             defaultHeight,
+             fullScreen: false,
+             maximize: false,
+         });
++        const blocker = await adblocker1.ElectronBlocker.fromLists(fetch,adblocker1.fullLists,
```

Comment:
Same as the previous review comment. Id rather not possibly be slapped with a takedown.